cmd/chart.go

```diff
@@ -81,9 +81,9 @@ func newChartMoveCmd() *cobra.Command {
 	f.UintVar(&retries, "retries", defaultRetries, "number of times to retry push operations")
 	f.StringVar(&output, "out", "*.relocated.tgz", "name of the resulting chart")
 
-	f.StringVar(&toArchive, "to-archive", "", "save the chart and all its dependencies to an intermediate archive tarball")
+	f.StringVar(&toArchive, "to-intermediate-bundle", "", "save the chart and all its dependencies to an intermediate archive tarball")
 
-	if err := cmd.Flags().MarkHidden("to-archive"); err != nil {
+	if err := cmd.Flags().MarkHidden("to-intermediate-bundle"); err != nil {
```

Comment:
maybe now it's time to make this flag visible since we have the e2e implemented no?

Reply:
will do